### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by pxcfbof**
## What does this PR do?
Add resource(s) 

### Description
Added a vim sheet that have more than 15 language translations. Also backed by duckDuckGo and should be a alternative in your list.

### Why is this valuable (or not)?
Easy to understand, and available in MANY languages.

### How do we know it's really free?
You can read about how to contribute (and help improve) by viewing our [README ](https://github.com/rtorr/vim-cheat-sheet/blob/master/README.md). There you can see how to set up this project, or how to contribute a new language. Here is a big thank you to our [contributors!](https://github.com/rtorr/vim-cheat-sheet/graphs/contributors)

This project is licensed under [The MIT License (MIT) ](https://github.com/rtorr/vim-cheat-sheet/blob/master/License.txt).

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -208,6 +208,7 @@
 * [Vim Avançado (pt)](https://aurelio.net/vim/vim-avancado.txt) - Aurelio Marinho Jargas (HTML)
 * [Vim Básico (pt)](https://aurelio.net/vim/vim-basico.txt) - Aurelio Marinho Jargas (HTML)
 * [Vim Cheat Sheet](https://www.cs.cmu.edu/~15131/f17/topics/vim/vim-cheatsheet.pdf) - Allison McKnight (PDF)
+* [Vim Cheat Sheet](https://vim.rtorr.com) - Many languages - Richard Torruellas(HTML)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:211]:
```suggestion
* [Vim Cheat Sheet](https://vim.rtorr.com) - Richard Torruellas(HTML)
```

Comment (pxcfbof) [reply more/free-programming-cheatsheets.md:211]: Changes done.

**Comment by pxcfbof** [comment]
I have made the echanges and evyerhing looks OK.

File: more/free-programming-cheatsheets.md (hunk 1 above)
Comment (eshellman) [inline more/free-programming-cheatsheets.md:211]:
```suggestion
* [Vim Cheat Sheet](https://vim.rtorr.com) - Richard Torruellas (HTML, multi-lingual)
```
nice resource.

**Comment by pxcfbof** [comment]
Conflict fixed.

**Comment by pxcfbof** [comment]
Conflict solved.

**Comment by eshellman** [comment]
no changes

**Pull request closed without merging**